frontend/device/android/device.lua

```diff
@@ -148,6 +151,20 @@ function Device:init()
                     UIManager:scheduleIn(0.2, function()
                         require("apps/reader/readerui"):doShowReader(new_file)
                     end)
+                else
+                    -- check if we're resuming from importing content.
+                    local content_path = android.getLastImportedPath()
```

Comment:
I don't get much of the Android code part :) so just asking: does this lastImportedPath/getLastImportedPath() gets reset back to null/nil after this, so you don't get stuck getting in this if when you resume koreader later?

Reply:
Sure, it is in https://github.com/koreader/android-luajit-launcher/pull/203/files#diff-272d98f2e8dcd4377fdca0ca2a53178aR193-R197